spec/controllers/idv/hybrid_mobile/capture_complete_controller_spec.rb

```diff
@@ -4,6 +4,16 @@
   include IdvHelper
 
   let(:user) { create(:user) }
+
+  let!(:document_capture_session) do
+    DocumentCaptureSession.create!(
```

Comment:
Does this spec require an actual DocumentCaptureSession in the db? I don't see it in use.

Reply:
Yes, the session is validated in a `before_action` (`check_valid_document_capture_session`)